ChaosMod/Effects/db/Vehs/VehsNoCollision.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+	Effect by Last0xygen
+*/
+
+#include <stdafx.h>
+
+static void OnTick()
+{
+	Ped player = PLAYER_PED_ID();
+	if (IS_PED_IN_ANY_VEHICLE(player, false))
+	{
+		Vehicle playerVeh = GET_VEHICLE_PED_IS_IN(player, false);
+		std::vector<Vehicle> foundVehs = GetNearbyVehicles(player, 50, 100);
+		std::vector<Ped> foundPeds = GetNearbyPeds(player, 50, 100);
+		for (Vehicle veh : foundVehs)
+		{
+			SET_ENTITY_NO_COLLISION_ENTITY(veh, playerVeh, true);
```

Comment:
Does this reset every frame? Or is there cleanup required?

Reply:
Every frame. False for the last parameter would be a lasting effect which i don’t know how to disable again 😅